src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/codeactions/LSPCodeActionAnnotator.java

```diff
@@ -132,9 +132,14 @@ protected Info doCollectInformation(PsiFile file, Editor editor) {
                     params.setContext(context);
                     params.setTextDocument(new TextDocumentIdentifier(LSPIJUtils.toUri(collectedInfo.file).toString()));
                     params.setRange(diagnostic.getRange());
-                    CompletableFuture<List<Either<Command, CodeAction>>> codeAction = entry.getKey().getInitializedServer().thenComposeAsync(server -> server.getTextDocumentService().codeAction(params));
+                    LanguageServerWrapper wrap = entry.getKey();
```

Comment:
Did you mean to use this variable somewhere? Maybe on the next line? I don't see any references.

Reply:
You're right, that was part of a log statement. I'll take it out.